### DISCUSSION TITLE
Server Error on demo.netbox.dev since v3.5

### DISCUSSION THREAD

**Discussion by dreng** [netbox-community/netbox#12386] [Q&A] [answered]
Currently the demo page is not fully accessible. You can see the home page at first, but after logging in a server error occours.

> <class 'KeyError'>
>
>'aef06f13-9f06-4442-aefe-30f67ad551d6'
>
>Python version: 3.8.10
>NetBox version: 3.5.0

Is this an ongoing process or broke something with the upgrade to v3.5?

**Comment by jeremystretch** [comment, marked as answer]
This was an oversight on my part: I neglected to update the demo instance to use demo data compatible with version 3.5. I'm rebuilding it now, and it should be working again shortly. Apologies for the inconvenience.
